episodes/06-raster-intro.md

```diff
@@ -19,7 +19,7 @@ exercises: 30
 
 Raster datasets have been introduced in [Episode 1: Introduction to Raster Data]({{site.baseurl}}/01-intro-raster-data). Here, we introduce the fundamental principles, packages and metadata/raster attributes for working with raster data in Python. We will also explore how Python handles missing and bad data values.
```

Comment:
```suggestion
Raster datasets have been introduced in [Episode 1: Introduction to Raster Data](01-intro-raster-data.md). Here, we introduce the fundamental principles, packages and metadata/raster attributes for working with raster data in Python. We will also explore how Python handles missing and bad data values.
```

Reply:
And there is another `site.baseurl` need to be changed in [line 30](https://github.com/esciencecenter-digital-skills/geospatial-python/pull/85/files#diff-8f8ae5a2f96ecae9746527ce8e01613d75574b6c41df656e1961d83d419b6cccL30)